sklearn/feature_extraction/hashing.py

```diff
@@ -170,3 +176,25 @@ def transform(self, raw_X):
         if self.non_negative:
             np.abs(X.data, X.data)
         return X
+
+    def get_feature_names(self):
+        """Returns OrderedDict of the feature mappings.
+
+        Returns
+        -------
+        feature_to_index_map : OrderedDict of the feature mappings.
```

Comment:
get_feature_names must return a list. Conventionally, it would be a list of strings.

Reply:
Alright, should it be a list of list of strings, or a list of comma separated strings?

Reply:
I wouldn't use something as common as comma. Newline or '|' perhaps.